pylint_odoo/checkers/no_modules.py

```diff
@@ -251,6 +257,23 @@
     # From odoo/odoo 10.0: odoo/odoo/fields.py:29
     'digits_compute:digits', 'select:index'
 ]
+FIELDS_METHOD = {
+    'Many2many': 4,
+    'One2many': 2,
+    'Many2one': 1,
+    'Reference': 1,
+    'Selection': 1,
+    'Binary': 0,
```

Comment:
You have the case with index 0 from a default.
You could avoid define it here. (like as my last comment about)